### PR TITLE
Adding RestoreZookeeper utility to accumulo admin command

### PR DESCRIPTION
**Pull request by cshannon**

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooReaderWriter.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public interface Mutator {
     byte[] mutate(byte[] currentValue) throws AcceptableThriftTableOperationException;
   }
 
-  public ZooReaderWriter(SiteConfiguration conf) {
+  public ZooReaderWriter(AccumuloConfiguration conf) {
```
Comment (EdColeman) [inline core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooReaderWriter.java:43]: Why is this change necessary?  I believe site configuration has the ZK info, I'm not sure that an AccumuloConfiguration must have it. It could be a subset. Or, maybe I have it backward?

Comment (EdColeman) [reply core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooReaderWriter.java:43]: You may be correct - it looks like https://github.com/apache/accumulo/pull/2215 wanted to get away from SiteConfiguration

**Comment by cshannon** [comment]
@milleruntime - Can this issue and #2815 be merged?
